docs/source/CONTRIBUTING.rst

```diff
@@ -185,15 +156,17 @@ Release roadmap
 The Fabric release roadmap is managed as a list of
 `GitHub issues with Epic label <https://github.com/hyperledger/fabric/labels/Epic>`__.
 
-Communications
-~~~~~~~~~~~~~~
+Communications and Getting Help
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We use the `Fabric mailing list <https://lists.hyperledger.org/g/fabric/>`__ for formal communication and
```

Comment:
I believe this requires a Linux Foundation ID to post messages, but the instructions to create one have been removed in this PR.  Should we move the instructions to this section?

Reply:
Ah, I forgot mailing list requires Linux Foundation ID, I'll re-add that section and point out the need for mailing list.

Reply:
LFID is distinct from Groups.IO

Reply:
@ryjones Ok, so the only reason most contributors would need a Linux Foundation account now is for the wiki?

Reply:
Wiki and Jira, but we're moving away from Jira

Reply:
Ok, so I removed the section about Getting a Linux Foundation ID and just mentioned it in passing instead:

> The Hyperledger Fabric [wiki](https://wiki.hyperledger.org/display/fabric) and the legacy issue management system in [Jira](https://jira.hyperledger.org/projects/FAB/issues) require a [Linux Foundation ID](https://identity.linuxfoundation.org/), but these resources are primarily for read-only reference and you will likely not need an ID.